### PR TITLE
Qingwei/fix test

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/LiveApi.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ export default class LiveApi {
     unresolvedPromises: Object;
     events: LiveEvents;
     onAuth: () => void;
+    state: Object;
```
Comment (borisyankov) [inline src/LiveApi.js]: Isn't it of type ApiState? :)

File: src/LiveApi.js [hunk 2]
```diff
@@ -197,25 +199,25 @@ export default class LiveApi {
         });
     }
 
-    send = function (callName: string, ...param: Object): ?LivePromise {
+    sendWithApiName = function (callName: string, ...param: Object): ?LivePromise {
```
Comment (borisyankov) [inline src/LiveApi.js]: Is that a good name? I don't understand what it mean.

Comment (borisyankov) [inline src/LiveApi.js]: BTW, these are public methods, you don't just rename public methods.

File: src/LiveApi.js [hunk 3]
```diff
@@ -232,4 +234,29 @@ export default class LiveApi {
...
+    send = (json: Object): ?LivePromise => {
+        const reqId = getUniqueId();
+        return this.sendRaw({
+            req_id: reqId,
+            ...json,
+        });
+    }
+
+    // TODO: should we deprecate this? preserve for backward compatibility
+    sendRaw = (json: Object): ?LivePromise => {
```
Comment (borisyankov) [inline src/LiveApi.js]: sendRaw is definitely needed.
Used by developers.binary.com and possibly some 3rd parties. (or not)

File: src/LiveApi.js (hunk 1 above)
Comment (qingweibinary) [inline src/LiveApi.js]: Yes, will add later

File: src/__tests__/unauthenticated-test.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ chai.use(chaiAsPromised);
 describe('unauthenticated', () => {
     let liveApi;
 
-    beforeEach(() => {
+    before(() => {
```
Comment (borisyankov) [inline src/__tests__/unauthenticated-test.js]: Why not beforeEach?

Comment (qingweibinary) [inline src/__tests__/unauthenticated-test.js]: Why beforeEach when before works?

Comment (borisyankov) [inline src/__tests__/unauthenticated-test.js]: before means it will reuse the connection.

**Pull request merged**